geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -235,6 +237,12 @@ AtomicLong5 getEventSeqNum() {
     return eventSeqNum;
   }
 
+  boolean notPrimary = true;
```

Comment:
This new member is mutated and accessed under different synchronization objects.